specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -267,8 +270,11 @@ model ImageCategoryPolicy {
   @doc("Image category.")
   category: ImageCategory;
   
-  @doc("Severity less than the threshold will be treated as accepted for the category. The value can only be -1, 0, 2, 4, 6. -1 means a maximum value, so that all severity can be treated as accepted.")
-  acceptLessThan: int32;
+  @doc("Severity less than the threshold will be treated as accepted for the category. The value can only be 2, 4, 6 for FourLevelSeverities.")
+  severityThreshold?: int32;
+  
+  @doc("The judgement kind to help server make judgement.")
+  judgementKind: JudgmentKind;
```

Comment:
Given `JudgementKind` has only 2 values and 1 requires that `severityThreshold` have a value (which itself is optional),

1. Why is `judgementKind` required even though `severityThreshold` is optional?
2. Given `AcceptAll` implies `severityThreshold` is absent (undefined) and `AcceptLessThan` requires a value, why not just use whether `severityThreshold` has a value and then treat it as `AcceptLessThan`?

This design requires more input - and has to be right, which isn't properly expressed here - than just using the existence and, if set, value of `severityThreshold`.

Reply:
Thanks for the feedback. If we use a `null` as for the threshold, user may get the information that no threshold is provided, so the API won't do judgement.

Reply:
If a threshold is required, then why did you author it as optional? It's odd - and probably not the best UX - to *require* users to author a `judgementKind` when they don't have to author a `severityThreshold`. Instead, maybe you should default `judgementKind` as it corresponds to an absent (different than a `null` value) `severityThreshold`.

Reply:
Just as offline discussed, we have got an alignment that severityThreshold could be null to represent accept all